src/styles/sections/section-footer.scss

```diff
@@ -0,0 +1,17 @@
+.footer {
+  width: 100%;
+  min-height: 200px;
+  background-image: url("./../images/footer.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position-x: center;
```

Comment:
set position `sticky` for footer, you will find more info about it in checklist.md